src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -74,16 +74,26 @@ const TitleBarWarning = (messageComponents) => {
   });
 };
 
+const TitleBarSpinner = (messageComponents) => {
+  return h(TitleBar, {
+    title: div({ role: 'alert', style: { display: 'flex', alignItems: 'center' } }, [
+      inlineSpinnerWithInnerStyles({ backgroundColor: colors.accent(0.35) }),
```

Comment:
At first glance, the spinner stuff here looks more complex than it needs to be. If you want an inline spinner, then `icon('loadingSpinner', ...)` might be adequate.

Reply:
Ah, thanks for the tip! I'll try that out.

Reply:
Simplified the code by just using the `spinner` component.